jetcd-core/src/main/java/io/etcd/jetcd/ClientConnectionManager.java

```diff
@@ -224,8 +225,14 @@ void close() {
             target = String.format(
                 "%s://%s/%s",
                 IPNameResolver.SCHEME,
-                builder.authority() != null ? builder.authority() : "",
-                endpoints.stream().map(e -> e.getHost() + ":" + e.getPort()).collect(Collectors.joining(",")));
+                builder.authority() != null ? builder.authority()
```

Comment:
I wonder if this is needed as if the authority is empty, then it can be computed automatically out of the URI in the IP resolver

Reply:
ok，i will make some changes to compute authority in Ip resolver

Reply:
I just change `e.getHost() + ":" + e.getPort()` to `URI::toString`, and then in Ip resolver we change re compute URI again.